src/test/java/games/strategy/engine/chat/ChatTest.java

```diff
@@ -27,7 +27,7 @@
 import games.strategy.util.ThreadUtil;
 
 public class ChatTest {
-  private static int SERVER_PORT = -1;
+  private int serverPort = -1;
```

Comment:
I converted the server port fields that were renamed in this PR to instance fields.  There appeared to be no reason why they were `static`.